evap/staff/tools.py

```diff
@@ -309,6 +309,13 @@ def merge_users(main_user, other_user, preview=False):
     return merged_user, errors, warnings
 
 
+def fill_infotext_formset(formset, formset_id, title_de="", title_en="", content_de="", content_en=""):
```

Comment:
To not mess up the ordering, the last four arguments could be kw-only:
```suggestion
def fill_infotext_formset(formset, formset_id, *, title_de="", title_en="", content_de="", content_en=""):
```
Furthermore, is the `formset_id` really the id of the formset, not a particular form in it?